answers.txt

```diff
@@ -1,8 +1,7 @@
-﻿Number of students who modified this file: 70 + 1 = 71 students
-
-Name of the last student who modified this file: John Davies

```

Comment:
I mistakenly put my oen name here at first, and not the name of the previous person who had updated the master.